packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -172,16 +194,18 @@ class ValidationResult {
       return '[✗]';
     else if (type == ValidationType.installed)
       return '[✓]';
-    else
-      return '[-]';
+    else // if (type == ValidationType.partial)
```

Comment:
can we use a switch instead?

Reply:
Done, but I'm not sure I like the way it turned out.  To avoid complaining about a missing return I had to add a default case, so it's logically equivalent to what was going on before.